src/pages/CreateAnalysisPage/CreateAnalysisPage.tsx

```diff
@@ -23,7 +23,7 @@ const CreateAnalysisPage = () => {
               </PageSection>
               :
               <PageSection className="page-section-no-results" variant={PageSectionVariants.light}>
-                <Error>{ hasSearched ? "No studies found" : "Search for a patient by entering their MRN above" } </Error>
+                <Error>{ hasSearched ? "No studies found for MRN #" + patientID : "Search for a patient by entering their MRN above" } </Error>
```

Comment:
Should add .toUpperCase() to patientID here given CNXC-338 (except for the search bar, every other instance of the MRN should probably be all caps).

Reply:
let's just leave this, no need to do it.